_kubernetes/03-administer-cluster.md

```diff
@@ -0,0 +1,138 @@
+---
+title: Administer Cluster
+order: 3
+duration: 5
+---
+
+## Set up Security Group
+
+If this is the first time you are connecting to the cluster, you need to create
+a security group to allow connections from your IP.
+
+1. Create a security group that allow connection to the cluster
+
+	```
+	openstack security group create kubernetes-api
+	openstack security group rule create --dst-port 6443 --remote-ip 101.116.123.53 kubernetes-api
+	```
+
+1. Apply the security group to the cluster. Use the following code snippet to
+   easily do this (replace `84091b19-fd32-4197-a92e-71838d0502c2` in the example
+with the ID of your cluster).
+
+	```
+	CLUSTER=84091b19-fd32-4197-a92e-71838d0502c2
+	STACK_ID=`openstack coe cluster show $CLUSTER -f value -c stack_id`
+	FLOATING_IP=`openstack stack output show $STACK_ID api_address -c output_value -f value`
+	PORT_ID=`openstack floating ip show $FLOATING_IP -c port_id -f value`
```

Comment:
Error:
```
justb@DOM-IMA-13ND0N2:~$ PORT_ID=`openstack floating ip show $FLOATING_IP -c port_id -f value`
'NoneType' object has no attribute 'keys'
```

this may or may not be a `openstack` version problem. My version is:
```
justb@DOM-IMA-13ND0N2:~$ os module list
...
| openstackclient | 3.14.2 |
...
```

Reply:
Try again with a new cluster? Due to the instability over last few days.